### PR TITLE
Sprint-1/issue-2/Create navigation menu

### PR DESCRIPTION
**Pull request by AnaCarrizales**
A navigation menu is made.

It has two options which are "Team 2" and its function is to show up the home screen. The other option is "Iniciar" and its function is to show the CRUD.

Style in "Iniciar" button has been changed.
Screenshot from 2024-01-24 17-58-17

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,24 @@
...
+
+ .navbar-collapse {
+    align-items: center;
+    justify-content: center;
+ }
+
+.link {
+    text-decoration: none;
+    color: rgba(0,0,0,.9);
+}
```
Comment (juperameza) [inline app/assets/stylesheets/application.css]: add a new line at the end of this file

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,24 @@
...
+body {
+}
+ .home {
+  max-height: 100vh;
+  min-height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #e6e6e6;
+  padding-bottom: 50px;
```
Comment (juperameza) [inline app/assets/stylesheets/application.css]: sort styles alphaneticallly

Comment (juperameza) [reply app/assets/stylesheets/application.css]: Select properties and do Ctrl + Shift + P to open the command palette and then search for "sort"

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<section class="home">
+<div class="container">
+<div class="row">
+<div class="col-lg-12 text-center">
+<h1 class="title">Sistema de Gestión de Eventos</h1>
+</div>
+
+</div>
+
+</section>
```
Comment (juperameza) [inline app/views/home/index.html.erb]: New line at the end of this file

File: app/controllers/planeadores_de_eventos_controller.rb
Comment (juperameza) [inline app/controllers/planeadores_de_eventos_controller.rb:1]: Why is this file here??? 
IN my opnion it doesnt have anything to do with the pr

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -5,11 +5,28 @@
...
+      </button>
+      <div class="collapse navbar-collapse" id="navbarText">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item active">
+          <%= link_to "Iniciar", { controller: "planeadores_de_eventos" }, class: "link" %>
+          </li>
+        </ul>
+      </div>
+      </div>
+    </nav>
```
Comment (juperameza) [inline app/views/layouts/application.html.erb]: Is beterr to use a partial

Comment (juperameza) [reply app/views/layouts/application.html.erb]: https://guides.rubyonrails.org/layouts_and_rendering.html

File: app/views/planeadores_de_eventos/_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+  <div>
+    <%= form.label :costo, style: "display: block" %>
+    <%= form.text_field :costo, class:"form-control" %>
+  </div>
+
+  <div>
+    <%= form.submit class:"btn btn-success", style:"margin-top:30px" %>
+  </div>
+<% end %>
```
Comment (juperameza) [inline app/views/planeadores_de_eventos/_form.html.erb]: Add a new line at the end of the file 
Also youre still using scaffold 
This isnt part of the scope of the pr youre describing

File: app/views/planeadores_de_eventos/edit.html.erb [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+<%= render "form", planeadores_de_evento: @planeadores_de_evento %>
+
+<br>
+
+<div>
+  <%= link_to "Ver evento individual", @planeadores_de_evento, class:"btn btn-primary" %> |
+  <%= link_to "Volver", planeadores_de_eventos_path, class:"btn btn-secondary" %>
+</div>
+</div>
```
Comment (juperameza) [inline app/views/planeadores_de_eventos/edit.html.erb]: Same here new line and the stuff about using scaffold

**Review by juperameza — CHANGES_REQUESTED** [review]

**Review by juperameza — APPROVED** [review]

**Review by ramondiaz21 — APPROVED** [review]
